ksqldb-metastore/src/main/java/io/confluent/ksql/metastore/model/StructuredDataSource.java

```diff
@@ -74,10 +73,6 @@
     this.ksqlTopic = requireNonNull(ksqlTopic, "ksqlTopic");
     this.casTarget = casTarget;
 
-    if (schema.valueContainsAny(SystemColumns.systemColumnNames())) {
```

Comment:
Same as above -- it'd be nice to have this change in a separate PR in order to limit this one to non-functional changes only.